pkg/runner/runner.go

```diff
@@ -122,6 +125,20 @@ func (r *runner) Run(parameters map[string]interface{}, name, namespace, kubecon
 	return jobStatus(inputDir.Path, ident)
 }
 
+func (r *runner) makeParameters(u *unstructured.Unstructured) map[string]interface{} {
+	s := u.Object["spec"]
+	spec, ok := s.(map[string]interface{})
+	if !ok {
+		logrus.Warnf("spec was not found")
+		spec = map[string]interface{}{}
+	}
+	parameters := paramconv.MapToSnake(spec)
+	parameters["meta"] = map[string]string{"namespace": u.GetNamespace(), "name": u.GetName()}
+	objectKey := fmt.Sprintf("_%v_%v", strings.Replace(r.GVK.Group, ".", "_", -1), strings.ToLower(r.GVK.Kind))
```

Comment:
Can you elaborate on what this is and why it's needed?

Reply:
The object key is for the playbook author to access the object. This sets the key to `_app_example_com_database`. This makes a particular key for each ansible operator that has a low chance of collision.